src/acvp_des.c

```diff
@@ -690,15 +692,18 @@ ACVP_RESULT acvp_des_kat_handler(ACVP_CTX *ctx, JSON_Object *obj)
 
                 /* Process the current DES encrypt test vector... */
                 rv = (cap->crypto_handler)(&tc);
+
                 if (rv != ACVP_SUCCESS) {
-                    ACVP_LOG_ERR("crypto module failed the operation");
-                    return ACVP_CRYPTO_MODULE_FAIL;
+                    if (rv != ACVP_CRYPTO_WRAP_FAIL) {
```

Comment:
We don't fail on wrap failure?

Reply:
It is an expected failure.  The server will send some decrypts that don't work and we need to return
"decryptFail" : true rather than the plaintext.